docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1271,6 +1271,44 @@ Spring Data Cloud Spanner publishes events extending the Spring Framework's `App
 |===
 
 
+=== Auditing
+
+Spring Data Cloud Spanner supports the `@LastModifiedDate` and `@LastModifiedBy` auditing annotations for properties:
```

Comment:
Maybe add a link to reference Spring Auditing documentation? This may be helpful to give more context on what AuditorAware is.

Reply:
There isn't a central Spring Data ref doc (only the actual implementations have them) ,but I can write more in this section.